docs/doc/01-cloud/20-bi-tool/00-metabase.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Connect Databend Cloud To Metabase
+sidebar_label: Metabase
+description: 
+  Connect Databend Cloud To Metabase
+---
+
+[Metabase](https://www.metabase.com/) is a simple and powerful analytics tool which lets anyone learn and make decisions from their company’s data—no technical knowledge required.
```

Comment:
Can we say “_[Metabase](https://www.metabase.com/) is a simple but powerful analytics tool that enables everyone to make business decisions using their data without requiring any technical skills._”？